doc/sphinx/language/gallina-extensions.rst

```diff
@@ -1982,17 +1986,22 @@ These are equivalent to a regular definition of `ident` followed by the declarat
 See also: more examples in user contribution category (Rocq/ALGEBRA).
 
 
-Print Canonical Projections.
-++++++++++++++++++++++++++++
+Printing Canonical Projections
+++++++++++++++++++++++++++++++
 
-This displays the list of global names that are components of some
-canonical structure. For each of them, the canonical structure of
-which it is a projection is indicated. For instance, the above example
-gives the following output:
+.. cmd:: Print Canonical Projections
 
-.. coqtop:: all
+   This displays the list of global names that are components of some
+   canonical structure. For each of them, the canonical structure of
+   which it is a projection is indicated.
+
+   .. example::
+
+   For instance, the above example gives the following output:
```

Comment:
This line is missing two indent spaces. Thank you GitLab for catching this :)

Reply:
Oh, sorry for that :) Just fixed.